mass_mailing_import_mailing_list/wizard/import_mailing_list.py

```diff
@@ -250,7 +251,10 @@ def parse_csv_data(self):
                 if self.import_type == 'adkd':
                     active_mail = row[headers['activemail']]
                 sokande_id = row[headers['sökande id']]
-                email = row[headers['e-postadress']]
+                if 'e-postadress' in headers:
+                    email = row[headers['e-postadress']]
+                else:
+                    email = ''
```

Comment:
The UserError on row 260 does not work
As the translation will not trigger on the final string it has to be written:
_('string {row}').format(row=row)